### ISSUE TITLE
Add Trackpad tab to Cesium Navigation Instructions

### ISSUE THREAD

**Issue by iannesbitt** [NCEAS/metacatui#2237]
**Describe the feature you'd like**
We have "Mouse" and "Touch" in the Navigation Instructions help dialog available with Cesium, but we need to add a tab with "Trackpad" instructions as well. At least instructions for "Pan", "Zoom", and "Rotate" with relevant gestures and keystrokes should be included.
I think this should be an easy addition, we just need to find relevant icons to include.

**Is your feature request related to a problem? Please describe.**
This came up when I was presenting during the DRP meeting and didn't have a mouse.

**Additional context**
![Screen Shot 2023-12-21 at 13 23 05](https://github.com/NCEAS/metacatui/assets/18689918/0564092e-071e-4ef0-b375-c02e5324cc03)

**Comment by robyngit** [comment]
Are touch and trackpad instructions not the same?

**Comment by iannesbitt** [comment]
No, for example I can't navigate the map like this with these actions on my Mac trackpad:
![Screen Shot 2024-01-02 at 10 39 57](https://github.com/NCEAS/metacatui/assets/18689918/0840dad2-af19-40e9-8eef-7f6bcdbde534)

Instead, pan view is a click+drag, zoom view is two finger drag same direction, tilt/rotate view is Ctrl+click+drag, and change camera angle is Shift+click+drag.
